client/web/src/api.ts

```diff
@@ -81,16 +81,19 @@ export function setUnraidCsrfToken(token?: string) {
 // incrementMetric hits the client metrics local API endpoint to
 // increment the given counter metric by one.
 export function incrementMetric(metricName: MetricName) {
-  const postData : MetricsPOSTData[] = [{
-    Name: metricName,
-    Type: "counter",
-    Value: 1
-  }]
+  const postData: MetricsPOSTData[] = [
```

Comment:
Should be the last update, sorry for the multiple pings

Changes in this file due to failing yarn format test https://github.com/tailscale/tailscale/actions/runs/7118200467/job/19380587422?pr=10484